### PR TITLE
Update app flags

### PR DESCRIPTION
**Pull request by spalmer25**
According to https://github.com/LedgerHQ/ledger-app-database/pull/157

### CODE REVIEW

**Comment by spalmer25** [comment]
Flags do not need to be updated

File: Makefile [hunk 1]
```diff
@@ -13,11 +13,28 @@ else ifeq ($(APP),tezos_wallet)
...
 endif
 
+ifeq ($(APP),tezos_baking)
+
 ifeq ($(TARGET_NAME), TARGET_NANOS)
-APP_LOAD_FLAGS=--appFlags 0x800  # APPLICATION_FLAG_LIBRARY
+# APPLICATION_FLAG_LIBRARY
+APP_LOAD_FLAGS=--appFlags 0x800
+else
+# APPLICATION_FLAG_LIBRARY + APPLICATION_FLAG_BOLOS_SETTINGS + APPLICATION_FLAG_GLOBAL_PIN
```
Comment (ajinkyaraj-23) [inline Makefile:22]: Check the wallet app flags. BOLOS_SETTINGS and GLOBAL_PIN are not required in nanosp,nanox and stax either.

Comment (spalmer25) [reply Makefile:22]: They are not required for the wallet app, but they are for the baking app
https://github.com/ajinkyaraj-23/ledger-app-database/blob/dfae81e7df6da75ea7e8bda119bb76fd3f256594/app-load-params-db.json#L1718

Comment (ajinkyaraj-23) [reply Makefile:22]: Actually, the app-database also needs to be changed. You can see my PR [here](https://github.com/LedgerHQ/ledger-app-database/pull/157). Raise similar PR change to tezos_baking in app-database. They generally approve it in one day.

**Review by ajinkyaraj-23 — APPROVED** [review]
LGTM.

**Pull request closed without merging**